app/views/unattended/report_templates/host_-_compare_content_hosts_packages.erb

```diff
@@ -15,41 +15,51 @@ template_inputs:
   value_type: plain
   resource_type: Katello::ActivationKey
 model: ReportTemplate
+require:
+- plugin: katello
+  version: 3.16.1
```

Comment:
We should add a migration to drop this template for users who don't have katello installed, otherwise it won't be updated when seeding (and in any case this template shouldn't be there for them).
You can check for something like
``` Foreman::Plugin.find('katello') ```

Reply:
added the migration for you.